modelz/aioclient.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+from typing import Any
+from http import HTTPStatus
+from urllib.parse import urljoin
+
+import aiohttp
+
+from rich.console import Console
+
+from .env import EnvConfig
+from .serde import Serde, SerdeEnum, TextSerde
+from modelz.utils import get_ssl_context_no_verify
+
+
+TIMEOUT = 300
+console = Console()
+DEFAULT_RESP_SERDE = TextSerde()
+DEFAULT_RETRY = 3
```

Comment:
never used?

Reply:
No, the 4 global variables are used.

Reply:
maybe we could remove DEFAULT_RETRY from modelz-py, since langchain will wrap everything in tenacity's retry decorators?

Reply:
Or we could use this https://pypi.org/project/aiohttp-retry/ ?

Reply:
`modelz-py` is the SDK of modelz, not the sub-library of langchain. The most important part is to make sure all the modelz-related functions should work.

Reply:
Makes sense. Will add retry support.

Reply:
Added retry support in ef8f5a6 and 230479a.